meshers/transvoxel/voxel_mesher_transvoxel.cpp

```diff
@@ -908,20 +908,20 @@ void VoxelMesherTransvoxel::build_transition(const VoxelBuffer &p_voxels, unsign
 void VoxelMesherTransvoxel::reset_reuse_cells(Vector3i block_size) {
 	_block_size = block_size;
 	unsigned int deck_area = block_size.x * block_size.y;
-	for (int i = 0; i < _cache.size(); ++i) {
+	for (unsigned int i = 0; i < _cache.size(); ++i) {
 		std::vector<ReuseCell> &deck = _cache[i];
 		deck.resize(deck_area);
-		for (int j = 0; j < deck.size(); ++j) {
+		for (long unsigned int j = 0; j < deck.size(); ++j) {
```

Comment:
Is GCC complaining about `long` too? `deck` will never have such a large size

Reply:
size() returns std::vector::size_type
gcc reported this:
```
modules/voxel/meshers/transvoxel/voxel_mesher_transvoxel.cpp: In member function 'void VoxelMesherTransvoxel::reset_reuse_cells(Vector3i)':
modules/voxel/meshers/transvoxel/voxel_mesher_transvoxel.cpp:914:21: warning: comparison of integer expressions of different signedness: 'int' and 'std::vector<VoxelMesherTransvoxel::ReuseCell>::size_type' {aka 'long unsigned int'} [-Wsign-compare]
```

Reply:
`size_t` or iterator would do then

Reply:
done and update pushed